src/style/color-palette-extended.css

```diff
@@ -165,8 +165,148 @@
     --color-glaucous-darker: 37, 71, 88; /* #254758 */
 }
 
+:root[data-theme="force-dark"] {
```

Comment:
I'm not sure if this strategy is gonna work for us. The reason is if you write a specific darkmode style for a limel-component, using `@media (prefers-color-scheme: dark)`, the styles will not get applied by fiddling the (🌙 dark / ☀️ light) toggle of Kompendium. Which means it won't work in the webclinet neither, if we do the same strategy there too. You can see the styles only if you change the dark/light using system preferences.

I hope this video shows what I mean.

https://user-images.githubusercontent.com/35954987/210054586-c2b1605d-9804-442d-ac37-93d3e4544ce9.mov

Not sure how to solve this. Do we need to consider adding some `:root[data-theme="force-dark"]` style as well as `@media (prefers-color-scheme: dark)`, and basically duplicate whatever styles we are writing for the dark mode?